src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -0,0 +1,3 @@
+export default function PlantingProgressList(): JSX.Element {
+  return <p>'I am a list'</p>;
```

Comment:
table code goes here

Reply:
@constanzauanini you and I can share this work for now.. replacing this with table etc.

Reply:
Sounds good!